README.md

```diff
@@ -73,8 +73,13 @@ in your executable path.
 With your [GOPATH](https://github.com/golang/go/wiki/GOPATH) already set up:
 
 ``` sh
-$ go get github.com/github/hub
-$ cd "$GOPATH"/src/github.com/github/hub
+$ mkdir -p "$GOPATH"/src/github.com/github
+$ cd "$GOPATH"/src/github.com/github
```

Comment:
Minor: could we clone into `"$GOPATH"/src/github.com/github/hub` directly and have a single `cd` command that puts us straight into the newly cloned directory, instead of having 2 separate `cd` operations?

`cd "$GOPATH"/src/github.com/github/hub`

Reply:
Sure!